examples/listbox/js/listbox.js

```diff
@@ -196,6 +218,13 @@ aria.Listbox.prototype.checkKeyPress = function (evt) {
         this.focusItem(nextUnselected);
       }
       break;
+    case 65:
+      // handle control + A
+      if (evt.ctrlKey) {
```

Comment:
```suggestion
      if (this.multiselectable && evt.ctrlKey) {
```

Reply:
Good catch!

Fixed this + the failing test :)